third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -48,7 +48,9 @@ PjRtComputationClient::PjRtComputationClient() {
     client_ = std::move(xla::GetCpuClient(/*asynchronous=*/false).ValueOrDie());
   } else if (device_type == "TPU") {
     TF_VLOG(1) << "Initializing PjRt TPU client...";
-    client_ = xla::GetTpuClient(/*max_inflight_computations=*/1).ValueOrDie();
+    int64_t max_inflight_computations = sys_util::GetEnvInt(
+        env::kEnvPjRtTpuMaxInflightComputations, /*defval=*/32);
```

Comment:
Why is 32 the right `defval`?

Reply:
I see you did this to match that of JAX. Does it make sense to verify PT/XLA will benefit from the same config or something slightly different?

Reply:
I copied the default value from JAX. I made it configurable, so we can try a few different values once we have a better set of benchmarks.

Reply:
Yeah, it'd great to track this as an internal issue to determine the optimal size for PyTorch/XLA.

CC @JackCaoG